src/components/DefaultRedirectionSnackbar/helpers.js

```diff
@@ -0,0 +1,90 @@
+import { deconstructRedirectLink, hasQueryBeenLoaded } from 'cozy-client'
+
+export const NAV_APP_SHOW_THRESHOLD = 3
+
+export const HOME_DEFAULT_REDIRECTION = 'home/'
+
+export const incrementNavAppShowCount = async (
+  client,
+  instanceSettingsResult
+) => {
+  const { nav_app_show_count } = instanceSettingsResult.data.attributes
+
+  const instanceSettings = {
+    data: {
+      ...instanceSettingsResult.data,
+      attributes: {
+        ...instanceSettingsResult.data.attributes,
+        nav_app_show_count: nav_app_show_count ? nav_app_show_count + 1 : 1
+      }
```

Comment:
Not very convinced to store that data to the `io.cozy.settings.instance` document. I think this should be stored in a `settings doctype` for the home app. Something like `io.cozy.home.settings`.

Reply:
Oh you're right it sounds better here.